pkg/utils/net/net.go

```diff
@@ -45,3 +49,33 @@ func GetRequestIP(req *http.Request) string {
 
 	return address
 }
+
+func GetData(request *restful.Request, response *restful.Response, url string) {
```

Comment:
no need to move function here, we probably not going to use it elsewhere, I think place it in `pkg/kapis/servicemesh/metrics/v1alpha2/handler.go` would be better

Reply:
done